ipatests/pytest_plugins/integration.py

```diff
@@ -30,9 +30,6 @@
 
 from ipapython import ipautil
 from ipapython.ipa_log_manager import log_mgr
-from ipatests.test_integration import tasks
-from ipatests.test_integration.config import Config
-from ipatests.test_integration.env_config import get_global_config
 
```

Comment:
tl;dr it breaks :)

It's a chicken and egg problem. ```ipatests.test_integration.__init__``` checks for ```pytest.config.getoption('ipaclient_unittests', False)```. On the other hand, pytests imports the plugin when it loads ```conftest.py```. At import time of ```conftest```, ```pytest.config``` is not available. The config namespace is created at a later stage.

We should clean this up at a later stage and move all dependencies of the integration plugin out of ```test_integration```. I tried it but it is a lot of work and requires heavy modifications of all integration tests. This approach is simpler and less risky.